pkg/parse/run_test.go

```diff
@@ -158,6 +159,8 @@ func TestRun(t *testing.T) {
 		name                       string
 		commit                     string
 		sourceRootExist            bool
+		hydrationEnabled           bool
```

Comment:
nit: Can we unify and call it `renderingEnabled`

Reply:
done